_pages/hiring-and-jobs/assignee-detail.md

```diff
@@ -10,126 +10,71 @@ questions:
+
+Internal hiring is done for three types of hiring processes: reassignment (transferring from one part of TTS to another), Excepted Service Merit Promotions, and details. 
+
+A reassignment is a lateral transfer to a new TTS office at the same GS level.
+
+An Excepted Service Merit Promotion is a permanent assignment to a new position description at a higher GS level. Review the [Merit Promotion Handbook page]({{site.baseurl}}/promotions/) for more information.
+
+A detail is a temporary assignment to a different position. Details last up to 120 days and can be extended up to another 120 days (NTE 240 days total), provided that:
+* The detailee, the TTS supervisor, the Business Unit Director, and the Assistant Commissioner agree, as well as the employee’s permanent supervisor.   
+* The employee is detailed at the same GS level they currently occupy. If the detailee has gone up in GS level, their detail must end at 120 days, due to federal policy and bargaining unit (union) contracts around temporary promotions. 
+
+## Internal Role Prioritization
+Internal roles do not need to be prioritized through Talent’s [formal prioritization process](https://docs.google.com/document/d/1V-7IyFIlLifgRg89TNKTS5oisOF-QdAZsWYCy7ot7AA/edit). However, the head of a Business Unit needs to approve the internal opportunity and add it to Talent’s Hiring Actions and Prioritization tracking spreadsheet. This ensures that Talent can track which actions have been approved.
+
+## Internal Job prep
+Internal postings are created using the following templates:
+* [TTSJobs Announcement Template](https://docs.google.com/document/d/1YIliZcF8dhqs4GzBAKYj5niqdgcN4tggTTDl3zeIOO8/edit)
+* [TTSJobs Application Form Template](https://docs.google.com/forms/d/1kpsTvpfCBox7ksQtX9VjQQdjkflPGCdKIZK-a9nUDPU/edit)
+* [TTS Interview Guide Template](https://docs.google.com/document/d/1GCVgMUBS-Hi7STLdWO6Ci47LXbYCz85U2kpVA71AZLc/edit)
+  * Best practices for developing an interview guide are available in the [Hiring Manager Guide](https://docs.google.com/document/d/1HDZYwc8E5_WepNvj55sqa3gc3W1NkHpd_110mSyD6RQ/edit#).
+  * An interview guide for the role may already exist if the role was previously posted; the Talent Team can check.
+  
+## Posting an Internal Role
+When the templates are ready for review, the Hiring Manager sends the announcement, application form, and interview guide to Talent at [jointts@gsa.gov](mailto:jointts@gsa.gov). A Talent team member will review. Once materials have been approved by Talent, they will add the posting to #tts-jobs on Slack and the [TTSJobs handbook page]({{site.baseurl}}/ttsjobs/). 
+
+## Internal Interviews 
+Hiring Managers select an interview panel and schedule interviews, the same as would happen for an external hire. Best practices are available in the [Hiring Manager Guide](https://docs.google.com/document/d/1HDZYwc8E5_WepNvj55sqa3gc3W1NkHpd_110mSyD6RQ/edit).
```

Comment:
Not relevant for the purposes of this review and it may have happened already and I'm not aware, but there should probably be some discussion about moving that guide out of google docs and into either the handbook materials for managers or the site that wg-hiring was working on.